responses/12-visualize-pipeline.md

```diff
@@ -5,7 +5,7 @@ Seeing the structure of a pipeline as a visual is powerful. Viewing connections
 Below is a makefile that is very similar to the one you have in your code repository (the option configurations and `source` calls were removed for brevity, but they are unchanged):
 ```r
 
-p1_fetch <- list(
+p1_targets_list <- list(
```

Comment:
Oh, thanks for catching this! I updated in `responses/40-datasci-conventions.md` but missed the changes needed here.